.circleci/src/jobs/move_release_notes.yml

```
@@ -0,0 +1,13 @@
+# Last step of a production delivery. Move all unreleased release notes to a new directory.
+docker:
+  - image: circleci/node:14.9.0
+resource_class: small
+steps:
+  - checkout
+  - prepare_workspace
+  - restore_environment_variables
+  - restore_yarn_tools_cache
+  - run:
+      name: Move release notes
+      command: tools/move-release-notes move-to ${NEW_VERSION_NAME} --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME} --branch ${CIRCLE_BRANCH}
+  #- notify TDB in LUN-30
```

With the following you can view all todos in intellij directly
```suggestion
  #- notify TODO in LUN-30
```